Game/data/ore_resource.gd

```diff
@@ -1,6 +1,8 @@
 class_name OreResource
 extends Resource
 
+## User facing icon to display in the game grid
+@export var icon: AtlasTexture = null
```

Comment:
Needed to add icons to all ores so that they could be displaye din the game grid.